hazelcast-spring/src/main/resources/hazelcast-spring-4.1.xsd

```diff
@@ -4937,6 +4938,57 @@
         </xs:attribute>
     </xs:complexType>
 
+    <xs:complexType name="sql">
+        <xs:annotation>
+            <xs:documentation>
+                SQL service configuration.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:all>
+            <xs:element name="executor-pool-size" type="xs:int" minOccurs="0" maxOccurs="1" default="-1">
+                <xs:annotation>
+                    <xs:documentation>
+                        Sets the number of threads responsible for query execution.
+                        The default value -1 sets the pool size equal to the number of CPU cores, and should be good enough
+                        for the most workloads.
+                        Setting the value to less than the number of CPU cores will limit the degree of parallelism of the SQL
+                        subsystem. This may be beneficial if you would like to prioritize other CPU-intensive workloads on the
+                        same machine.
+                        It is not recommended to set the value of this parameter greater than the number of CPU cores because it
+                        may decrease the system's overall performance due to excessive context switches.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="operation-pool-size" type="xs:int" minOccurs="0" maxOccurs="1" default="-1">
+                <xs:annotation>
+                    <xs:documentation>
+                        Sets the number of threads responsible for network operations processing.
+                        When Hazelcast members execute a query, they send commands to each other over the network to coordinate
+                        the execution. This includes requests to start or stop query execution, or a request to process a batch
+                        of data. These commands are processed in a separate operation thread pool, to avoid frequent interruption
+                        of running query fragments.
+                        The default value -1 sets the pool size equal to the number of CPU cores, and should be good enough
+                        for the most workloads.
+                        Setting the value to less than the number of CPU cores may improve the overall performance on machines
+                        with large CPU count, because it will decrease the number of context switches.
+                        It is not recommended to set the value of this parameter greater than the number of CPU cores because it
+                        may decrease the system's overall performance due to excessive context switches.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="query-timeout-millis" type="xs:unsignedLong" minOccurs="0" maxOccurs="1">
```

Comment:
What is the default value of this parameter? Shall we add it to the schema definition?

Reply:
Fixed in both XSD files.